server/tests/api/videos/videos-common-filters.ts

```diff
@@ -70,10 +76,18 @@ describe('Test videos filter', function () {
 
   describe('Check deprecated videos filter', function () {
 
-    async function getVideosNames (server: PeerTubeServer, token: string, filter: string, expectedStatus = HttpStatusCode.OK_200) {
+    async function getVideosNames (
```

Comment:
Please convert all args as an object

When you reach 3 or 4 arguments it means it's time to switch to the object arg style